src/stylesheets/app.scss

```diff
@@ -1,5 +1,5 @@
 @import "../../node_modules/bootstrap/dist/css/bootstrap.css";
-@import "../../node_modules/opds-web-client/lib/stylesheets/app.scss";
```

Comment:
Removing `panel.scss` was fine but why did you remove this file? It's still needed... and the new file does not replace this.

Reply:
I'll put it back in just in case, but I'm pretty sure this repo isn't actually using anything from it (I think I had just copy/pasted it from circ-web when I was first setting this up)...

Reply:
The repo is using a few mixins from that repo, unfortunately. If you remove it and try to build, it'll give you that warning.